README.md

```diff
@@ -267,64 +268,147 @@ standard JS template literal syntax:
+- `on` (`import { on } from '@ember/modifier';`)
+- `Input` (`import { Input } from '@ember/component';`)
+- `LinkTo` (`import { LinkTo } from '@ember/routing';`)
+- `TextArea` (`import { TextArea } from '@ember/component';`)
 
 [rfc-496]: https://github.com/emberjs/rfcs/pull/496
 
-Compatibility
-------------------------------------------------------------------------------
+## Compatibility
 
-* Ember.js v3.27 or above
-* Ember CLI v2.13 or above
-* `ember-cli-htmlbars` 6.0 or above
-* Node.js v12 or above
+- Ember.js v3.27 or above
+- Ember CLI v2.13 or above
+- `ember-cli-htmlbars` 6.0 or above
+- Node.js v12 or above
 
-
-Installation
-------------------------------------------------------------------------------
+## Installation
 
 ```
 ember install ember-template-imports
 ```
 
-Editor Integrations
-------------------------------------------------------------------------------
+## Usage for V2 Addons
+
+V2 addons are build by default with rollup, see
+[blueprint](https://github.com/embroider-build/addon-blueprint). This addon
+provides integrations to rollup to understand and transpile `.gjs` and `.gts`
+files
+
+### Step 1: Install babel plugin
+
+Add the `ember-template-imports` babel plugin to your babel config:
+
+```diff
+'use strict';
+
+module.exports = {
+  presets: ['@babel/preset-typescript'],
+  plugins: [
++    'ember-template-imports/babel-plugin',
+    '@embroider/addon-dev/template-colocation-plugin',
+    ['@babel/plugin-proposal-decorators', { legacy: true }],
+    '@babel/plugin-proposal-class-properties'
+  ]
+};
+```
+
+### Step 2: Understand and Transpile `<emplate>`
+
+The first step is add the `ember-template-imports` rollup plugin, which will
+understand the `<template>` tag `gjs` and `gts` files and transpiles it to an
+intermediate format:
+
+```js
+import { Addon } from '@embroider/addon-dev/rollup';
+import emberTemplateImports from 'ember-template-imports/rollup-plugin';
+
+const addon = new Addon({
+  srcDir: 'src',
+  destDir: 'dist'
+});
+
+export default {
+  // This provides defaults that work well alongside `publicEntrypoints` below.
+  // You can augment this if you need to.
+  output: addon.output(),
+
+  plugins: [
+    // These are the modules that users should be able to import from your
+    // addon. Anything not listed here may get optimized away.
+    addon.publicEntrypoints(['components/**/*.js', 'index.js'<% if (typescript) {%>, 'template-registry.js'<% } %>]),
+
+    // These are the modules that should get reexported into the traditional
+    // "app" tree. Things in here should also be in publicEntrypoints above, but
+    // not everything in publicEntrypoints necessarily needs to go here.
+    addon.appReexports(['components/**/*.js']),
+
+    // Follow the V2 Addon rules about dependencies. Your code can import from
+    // `dependencies` and `peerDependencies` as well as standard Ember-provided
+    // package names.
+    addon.dependencies(),
+
+    emberTemplateImports(),
```

Comment:
after seeing how this looks, and with the goal of v2 addons and template-imports being polaris-y, I _feel like_ the rollup plugin should live in `@embroider/addon-dev`.

I'd like to see:
```js
// existing
addon.hbs();

// proposed (in addition to the above):
addon.gjs();
```

Reply:
I'll bring this up with the embroider office hours group.

I don't think it'd need to block though.
TBH, addon-dev is already "convenience" apis, because making a package (even outside of ember) is quite cumbersome.
addon-dev can re-export these plugins.